### ISSUE TITLE
[Tables] Add clear filter button to Table View column filters

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#1989]
When filtering a table view by column, it would be very nice to have a "x" button in the input area to allow that filter input value to be cleared with a single click. Prior examples exist in the main search input, and in the filter control displayed while editing objects that utilize the Edit interface, such as Display Layouts.

**Comment by PunkChameleon** [comment]
@charlesh88 @dtailor90 Should this be closed?

**Comment by dtailor90** [comment]
@PunkChameleon - There is still some styling that needs to be done, I will close it once it is complete.

**Comment by charlesh88** [comment]
### Testing notes

1. View a table (that has data) that includes filtering, such as that for a telemetry element, or a Telemetry Table. 
1. Type a string into any column's filtering input. Observe: an "X" button should appear inside the input, on the right-hand side.
1. Click the "X" button. The entered text should be removed, and any column filtering will be removed.
